docs/DeveloperGuide.md

```diff
@@ -119,41 +119,98 @@ testers are expected to do more *exploratory* testing.</div>
-   Expected: No recipe is deleted. Console output tells you to input a number within the range of the recipe list numbers.
+5. **Test case**: `delete`<br>
+   **Expected**: No recipe is deleted. Console output tells you to use a index parameter for the delete function.
+   <br>*Example output*: The delete function takes in one parameter: {index}
+   <br>Input Example: delete 1
 
 ### Finding a recipe by keyword
 
-1. Prerequisites: List all recipes using the `list` command. Multiple recipes in the list. At least one with "soup" in recipe name.
+1. **Prerequisites**: List all recipes using the `list` command. Multiple recipes in the list. At least one with "soup" in recipe name.
 
-2. Test case: `find kw soup`<br>
-   Expected: Console output shows you which recipes have the word "soup" in its name.
+2. **Test case**: `find kw soup`<br>
+   **Expected**: Console output shows you which recipes have the word "soup" in its name.
+   <br>*Example output*: Here are your matches with keyword: soup
+   <br>soup / added on 2024-03-31 / url: www.food.com
 
-3. Test case: `find kw sou`<br>
-   Expected: No recipe is found. Console output tells you to check if your parameter is a word.
+3. **Test case**: `find kw sou`<br>
+   **Expected**: No recipe is found. Console output tells you to check if your parameter is a word.
+   <br>*Example output*: There were no matches. Try searching for something else.
 
-4. Test case: `find kw`<br>
-   Expected: Console output tells you to check that you inputted two arguments to the find method.
+4. **Test case**: `find kw`<br>
+   **Expected**: Console output tells you to check that you inputted two arguments to the find method.
+   <br>*Example output*: The find function accepts two parameters: {type} and {criteria}
+   <br>Input Example: find kw pizza
+   <br>Input Example: find date 2024-03-28
+
+5. **Test case**: `find kw 1`<br>
+   **Expected**: Console output tells you to ensure the keyword is an alphabet
+   <br>*Example output*: Parameter cannot be parsed as an word.
+   <br> Please enter a word using lower and upper case alphabets.
 
 ### Finding a recipe by date
 
-1. Prerequisites: List all recipes using the `list` command. Multiple recipes in the list. At least one with date as 2024-03-30.
+1. **Prerequisites**: List all recipes using the `list` command. Multiple recipes in the list. At least one with date as 2024-03-30.
+
+2. **Test case**: `find date 2024-03-30`<br>
+   **Expected**: Console output shows you which recipes were added on 2024-03-30.
+   <br>*Example output*: Here are your matches with date: 2024-03-30
+   <br>pizza / added on 2024-03-30 / url: www.food.com
 
-2. Test case: `find date 2024-03-30`<br>
-   Expected: Console output shows you which recipes were added on 2024-03-30.
+3. **Test case**: `find date 2025-03-03`<br>
+      **Expected**: Console output shows you that no recipes were added on 2025-03-03.
+      <br>*Example output*: There were no matches. Try searching for something else.
```

Comment:
The prerequite says nothing about whether a recipe with date 2025-03-03 is added. So, you can't conclude that this is the expected output.

You may want to reconsider having this test case. If you feel it's important, you need to modify the prerequisite.